hibp.js

```diff
@@ -139,10 +141,11 @@ const HIBP = {
 
   filterOutUnsafeBreaches(breaches) {
     return breaches.filter(
-      breach => breach.IsVerified &&
-                !breach.IsRetired &&
+      breach => !breach.IsRetired &&
                 !breach.IsSensitive &&
-                !breach.IsSpamList
+                !breach.IsSpamList &&
+                !breach.IsFabricated &&
```

Comment:
Similarly, this logic changed to add the `!breach.IsFabricated` check, but removed the `.IsVerified` check.

Reply:
Added .IsVerified check back in.

Reply:
Cool, I think this makes sense. We lost the `!breach.IsSensitive` check here, but I see it doing the filter above on L127 (if we aren't checking `includeSensitive` branch in `getBreachesForEmail()`).